src/shared/Core/HttpClientFactory.cs

```diff
@@ -197,6 +197,40 @@ public HttpClient CreateClient()
 #endif
             }
 
+            _trace.WriteLine($"Custom cookie file has been enabled with {_settings.CustomCookieFilePath}");
+            // If IsUsingCookieEnabled is enabled, set Cookie header from cookie file, which is written by libcurl
+            if (!string.IsNullOrWhiteSpace(_settings.CustomCookieFilePath))
+            {
+                // get the filename from gitconfig
+                string cookieFilePath = _settings.CustomCookieFilePath;
+                _trace.WriteLine($"Custom cookie file has been enabled with {cookieFilePath}");
+                
+                // Throw exception if cookie file not found
```

Comment:
Does Git also `die`/error if the configured cookie file is not found? Or just warn? We should do the same.

Reply:
As far as I could confirm by operating it, Git did not die.
Therefore, if the configured cookie file is not found, I will change it so that only operations related to cookies are not performed.